File: src/main/java/com/p2p/lending/service/CertificationService.java
package com.p2p.lending.service;

import java.util.Map;

import com.p2p.lending.entity.Certification;

public interface CertificationService {
	public String selectM(Integer uid);
	public boolean updateM(Map<String, Object> map);
	public boolean undate(Map<String, Object> map);
	public boolean upm(Map<String, Object> map); 
	public int insert(Certification cer);
	public Certification select(Integer uid);
}

File: src/main/java/com/p2p/lending/service/CertificationService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Certification
import org.springframework.stereotype.Service

@Service
interface CertificationService {
    fun selectM(uid: Int?): String
    fun updateM(map: Map<String, Any>): Boolean
    fun undate(map: Map<String, Any>): Boolean
    fun upm(map: Map<String, Any>): Boolean
    fun insert(cer: Certification): Int
    fun select(uid: Int?): Certification
}
